packages/framework/collabspaces/src/collabSpaces.ts

```diff
@@ -502,9 +531,35 @@ export class TempCollabSpaceRuntime
 	public async getCellDebugInfo(
 		row: number,
 		col: number,
-	): Promise<{ channel?: ICollabChannelCore }> {
+	): Promise<{ channel?: ICollabChannelCore; channelId?: string; rowId: string; colId: string }> {
 		const result = this.getCellInfo(row, col);
-		return { channel: await result.channel };
+		assert(result.channelId !== undefined, "channelId is missing");
```

Comment:
This would assert if called for a cell that has no channel. From API name / semantics it does not make much sense.